fixitfelix/fix.py

```diff
@@ -109,38 +111,37 @@ def write_chunks_to_file(
         tdms_writer.write_segment([new_channel])
 
 
-def export_correct_data(
-    tdms_path: pathlib.Path, meta: source.MetaData, export_path: pathlib.Path
-) -> None:
-    """Exports the valid data slices into a new TDMS file on disk.
-
-    Before the export is done, all input parameters are checked for consistency are checked.
-    Moreover, the function raises an execption if MetaData and TdmsFile do not match. (WIP)
-
-    Don't mind the nested for loops, the sizes of the iterators of the first
-    and second stage are very small.
+def preprocess(meta: source.MetaData, path: pathlib.Path) -> Any:
+    """Runs all consistency checks on given tdms file and meta data. All input parameters are checked for consistency.
```

Comment:
We do not have a CI in github, but i think we should limit line lengths to 80.

Maybe use black to lint the code.

Reply:
Good idea. I'm gonna use black in the future.